proto/schema/v1/genesis.proto

```diff
@@ -3,12 +3,15 @@ package sonrio.sonr.schema;
 
 import "gogoproto/gogo.proto";
 import "schema/v1/params.proto";
+import "schema/v1/what_is.proto";
 // this line is used by starport scaffolding # genesis/proto/import
 
 option go_package = "github.com/sonr-io/sonr/x/schema/types";
 
 // GenesisState defines the schema module's genesis state.
 message GenesisState {
   Params params = 1 [(gogoproto.nullable) = false];
+  repeated WhatIs whatIsList = 2 [(gogoproto.nullable) = false];
```

Comment:
In protobuf you need to use underscore_case. This makes sure that protobuf will generate the target code with correct conventions.

Reply:
+1

Reply:
Changed